src/main.rs

```diff
@@ -199,7 +225,12 @@ impl State {
 
         // Self-sign the certificate.
         let crt = tbs.sign(&pki)?;
-        Ok(Self { key, crt, san })
+        Ok(Self {
+            key,
+            crt,
+            san,
+            config: ConfigurationFile::default(),
+        })
```

Comment:
let config = Default::default();
```suggestion
        Ok(Self {
            key,
            crt,
            san,
            config,
        })
```

Reply:
Fixed, I think.